src/Grpc.AspNetCore.Server/IGrpcServiceActivator.cs

```diff
@@ -0,0 +1,26 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+namespace Grpc.AspNetCore.Server
+{
+    public interface IGrpcServiceActivator<TGrpcService> where TGrpcService : class
```

Comment:
I would think the answer is yes, all public APIs must have doc comments. But as far as when they should be put in, that's another question. It might make sense at some point to have an API review, and once the API surface is roughly settled, we can do a doc pass.